spi-tester/src/main/java/org/ehcache/spi/test/ValidSPITesterException.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ehcache.spi.test;
+
+/**
+ * Can be used to report valid exceptions in SPI test so that the said exceptions don't cause a failure
+ * 
+ * @author rism
+ */
+public class ValidSPITesterException extends Exception {
```

Comment:
better to double check with a native english speaker but using 'valid' for an exception sounds a bit strange